travis/docker-push-setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This file is in a "travis" dir while others are in ".travis".
Lets try to stick to 1 name :)

Reply:
They are slightly different the ".travis" are duplicated per docker repository where as "travis" is once in the whole repo.

I'm happy to change travis to .travis but it just was remaining consistent with the current layout

Reply:
They all just contain little bash scripts for travis use only, lets call them all .travis :)